src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/WeekNum.txt

```diff
@@ -11525,76 +11525,76 @@
 
 // **************** SAME DAY - ALL HOURS TESTCASES  ****************
 
->> WeekNum(DateTimeValue("1/7/2021 0:00:00:00"),15)
+>> WeekNum(DateTimeValue("1/7/2021 0:00:00"),15)
```

Comment:
Instead of using the DateTimeValue, we should use the DateTime (or Date) function in tests. The former is locale-dependent, and we want the tests to be run in most locations. For example, that value could be interpreted both as January 7th and July 1st depending on whether the tests are run. If we use the Date[Time] function, the result will not be ambiguous.